platform/android/java/plugins/godotpaymentv3/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="org.godotengine.godot.plugin.paymentv3">
```

Comment:
In the other `AndroidManifest.xml` we seem to use only one indent level (4 spaces) for continuations, this should be kept consistent (one way or the other, I don't have a strict preference).